cmd/neofs-adm/internal/modules/morph/dump_hashes.go

```diff
@@ -152,7 +152,12 @@ func dumpCustomZoneHashes(cmd *cobra.Command, nnsHash util.Uint160, zone string,
 		})
 	}
 
-	sessionID, iter, err := unwrap.SessionIterator(inv.Call(nnsHash, "tokens"))
+	script, err := smartcontract.CreateCallAndPrefetchIteratorScript(nnsHash, "tokens", nnsMaxTokens)
+	if err != nil {
+		return fmt.Errorf("building prefetching script: %w", err)
```

Comment:
i wouldnt understand this msg if i saw it, if it's user-facing, suggest to clarify

Reply:
i do not know what to add here. a dev can understand it, a user that does not want to understand it, will not understand any info about building scripts. that an optimization, not sure it can be more clear